optuna/visualization/matplotlib/_parallel_coordinate.py

```diff
@@ -218,7 +218,6 @@ def _target(t: FrozenTrial) -> float:
         ax2.spines["top"].set_visible(False)
         ax2.spines["bottom"].set_visible(False)
         ax2.xaxis.set_visible(False)
-        ax2.plot([1] * len(param_values[i]), param_values[i], visible=False)
```

Comment:
In my understanding, no because this line does something strange. When we replace the line with `ax2.plot([1] * len(param_values[i]), param_values[i], markersize=17, color="red")` to make the plot visible on the current master branch, the plot method shows 

<img width="431" alt="Screenshot 2022-05-18 at 20 10 46" src="https://user-images.githubusercontent.com/7121753/169025927-d5adb621-06ef-42b7-bc49-27c25b064640.png">

Note that `red` vertical line is corresponding to the deleted line. As we confirmed, we have the vertical line without deleted line.

---

Code: 

```python
import optuna

def objective(trial):
    x = trial.suggest_float("x", -100, 100)
    y = trial.suggest_categorical("y", [-1, 0, 1])
    z = trial.suggest_categorical("z", [1])
    return x ** 2 + y + z


sampler = optuna.samplers.TPESampler(seed=10)
study = optuna.create_study(sampler=sampler)
study.optimize(objective, n_trials=10)
```